.changeset/hot-rules-tickle.md

```diff
@@ -0,0 +1,6 @@
+---
+'@vercel/build-utils': minor
+'vercel': minor
+---
+
+Output "diagnostics" directory during `vc build`
```

Comment:
It would be great to have some tests around this logic

Reply:
@TooTallNate Could you help with setting up some test?